gpii/node_modules/solutionsRegistry/src/schemas/solution-schema.json5

```diff
@@ -49,7 +49,14 @@
                         "type": "object",
                         "additionalProperties": {
                             "type": "object",
-                            "additionalProperties": false,
+                            "additionalProperties": {
+                                "properties": {
+                                    "label": {
```

Comment:
@the-t-in-rtf - do you think this is reasonable? I can't quite figure out what is going on here, the only value this "label" seems to take is "bitmask"

Reply:
@amb26, this is almost certainly incorrect.  It's saying that in addition to any supported settings described, you may also have one or more objects that include an optional label.  As this optional restriction is the only rule for those additional properties. it opens to door for any object at all to be allowed, including those that lack the required `schema` property.

If the intent is to allow a label within a setting, then the `label` property should be moved to the same level as the `schema` property, and the nested `additionalProperties` block should be collapsed/removed.

Reply:
You could also argue for being a lot more specific about what's allowed for `label`, i.e. a single element enum with `bitmask` as the only value.

Reply:
@amb26 for updating that schema I followed this suggestions https://github.com/GPII/universal/pull/821#discussion_r339559054 given by @the-t-in-rtf .